cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from cfme import test_requirements
```

Comment:
**REQUIRED**
Please put one blank between external and internal imports.

Reply:
added.